releases/release-1.26/release-notes/maps/pr-112607-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 112607
+releasenote:
+  text: Consider only plugin directory and not entire kubelet root when cleaning up
```

Comment:
past tense

`Considered only plugin directory and not entire kubelet root when cleaning up`

Reply:
Using `considered` here actually makes the readability lot more complicated.  let me change this to `volume mount cleanup now considers only plugin directory and not the entire kubelet root`